src/lexers/javascript-lexer.js

```diff
@@ -57,8 +58,45 @@ export default class JavascriptLexer extends BaseLexer {
 
             that.keys.push(entry)
           }
+        },
+        JSXElement(node) {
```

Comment:
This is the trap for JSX Elements.

Check if it's "Trans" --> Set the key --> Set the defaultContent --> If the set key is falsey and there is defaultContent, set the key to defaultContent --> if we now have a key, push to the keys array.

Check if it's "Interpolate" --> Set the key --> if we have a key, push to the keys array.